File: atomlessons/src/app/models/lesson.ts
export class Lesson {
    Title: string;
    Description: string;
    InstructorName: string;
    InstructorPhotoUrl: string;
    SubjectPhotoUrl: string;
    Time: Date;
    LessonEndTime: Date;

    constructor(title: string, description: string, instructor: string,
        instructorPhoto: string, subjectPhoto: string,
        lessonDate: Date
    ) {
        const HOUR_IN_MILLISECONDS = 3600000;

        this.Title = title;
        this.Description = description;
        this.InstructorName = instructor;
        this.InstructorPhotoUrl = instructorPhoto;
        this.SubjectPhotoUrl = subjectPhoto;
        this.Time = lessonDate;
        this.LessonEndTime = new Date();
        this.LessonEndTime.setTime(new Date(this.Time).getTime() + HOUR_IN_MILLISECONDS);
    }
}

File: AtomLessonsAPI/AtomLessonsAPI.DTO/Lesson.cs
using System;

namespace AtomLessonsAPI.DTO
{
    public class Lesson
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public string InstructorName { get; set; }
        public string InstructorPhotoUrl { get; set; }
        public string SubjectPhotoUrl { get; set; }
        public DateTime Time { get; set; }
    }
}
